### DISCUSSION TITLE
Zed Needs to be new Atom, but fast!

### DISCUSSION THREAD

**Discussion by ghost** [zed-industries/zed#8844] [Show and tell]
Atom has an impeccable design, as well as interesting features. The only problem with Atom was its performance (and some difficulties with LSP in some languages). However, now with the speed of Rust combined with GPUI, together with the creators of Atom, modeling Zed to be the "new Atom, only better" would be a great idea.
When I say this, I'm talking mainly in relation to design, reusing or improving the design of Atom elements (which, in my opinion, is the most beautiful editor ever created, along with Sublime Text).
I know there are open source branches of Atom, but none compare to the potential that Zed has - large community, special team, performance and features.

![images (10)](https://github.com/zed-industries/zed/assets/161986636/5ff4a441-3ac4-472d-9796-99f4e0ee7ec5)

**Comment by Moshyfawn** [comment]
I think you'll want to be part of these conversations:
- #8499
- #8763
